www/front_src/src/Resources/Filter/useFilter.ts

```diff
@@ -84,23 +89,21 @@ const useFilter = (): FilterState => {
     request: listCustomFilters,
   });
 
-  const getDefaultCriterias = (): Array<Criteria> =>
-    getDefaultFilter().criterias;
-
-  const getDefaultSearchCriteria = (): Criteria =>
-    getDefaultCriterias().find(propEq('name', 'search')) as Criteria;
-
   const [customFilters, setCustomFilters] = React.useState<Array<Filter>>([]);
-  const [filter, setFilter] = React.useState(getDefaultFilter());
-  const [nextSearch, setNextSearch] = React.useState<string | undefined>(
-    getDefaultSearchCriteria().value as string,
-  );
-  const [filterExpanded, setFilterExpanded] = React.useState(
-    getDefaultFilterExpanded(),
-  );
+  const [currentFilter, setCurrentFilter] = React.useState(getDefaultFilter());
+  const [appliedFilter, setAppliedFilter] = React.useState(getDefaultFilter());
```

Comment:
If I understand correctly, the applied filter is the current filter when the user press the "Enter" key, right ?

Reply:
Yes, or if a Filter is selected; or the Search button is pressed